package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "arkdelegates",
+  "version": "0.3.0",
+  "main": "index.js",
+  "repository": "git@github.com:roks0n/arkdelegates.git",
+  "author": "roks0n <haluzan.rok@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack --mode development",
+    "watch": "webpack --mode development --watch",
+    "build:production": "webpack --mode production"
+  },
+  "dependencies": {
+    "babel-preset-env": "^1.6.1",
```

Comment:
I would love to see these requirements being forced to the exact version. Frontend libraries are very unstable and are no one can rely on them being backwards compliant even if they are minor versions. Talking from experience.